src/NServiceBus.Persistence.DynamoDB.PersistenceTests/SetupFixture.cs

```diff
@@ -17,37 +17,38 @@ public async Task OneTimeSetUp()
         {
             DynamoDBClient = ClientFactory.CreateDynamoDBClient();
 
-            OutboxConfiguration = new OutboxPersistenceConfiguration()
+            OutboxTable = new TableConfiguration()
             {
-                TableName = $"{DateTime.UtcNow.Ticks}_{Path.GetFileNameWithoutExtension(Path.GetTempFileName())}_Outbox",
+                TableName =
+                    $"{DateTime.UtcNow.Ticks}_{Path.GetFileNameWithoutExtension(Path.GetTempFileName())}_Outbox",
                 TimeToLiveAttributeName = Guid.NewGuid().ToString("N") + "TTL",
                 PartitionKeyName = Guid.NewGuid().ToString("N") + "PK",
                 SortKeyName = Guid.NewGuid().ToString("N") + "SK",
-                TimeToLive = TimeSpan.FromSeconds(100)
             };
-            SagaConfiguration = new SagaPersistenceConfiguration()
+            SagaTable = new TableConfiguration()
             {
-                TableName = $"{DateTime.UtcNow.Ticks}_{Path.GetFileNameWithoutExtension(Path.GetTempFileName())}_Saga",
+                TableName =
```

Comment:
I know it was like that before but it got me wondering whether we should use the same ticks and temp file name across the run?